scripts/versions.gradle

```diff
@@ -45,6 +45,11 @@ versions.logging_interceptor = "4.9.2"
 versions.snowplow = "2.2.1"
 versions.snowplow_lifecycle = "2.2.0"
 
+versions.crypto = "1.1.0-alpha03"
+
+versions.sqlcipher = "4.5.0"
+versions.sqlire = "2.1.0"
```

Comment:
variable name should be sqlite instead of sqlire.

Reply:
Completed